dat/outfits/launchers/enygma_systems_turreted_fury_launcher.xml

```diff
@@ -5,13 +5,14 @@
   <size>small</size>
   <mass>25</mass>
   <price>80000</price>
-  <description>Enygma Systems has developed a turret mounted Fury launcher. It comes at a steep price, but can be a powerful means of discouraging pirate fighters.</description>
+  <description>The Fury class interceptor missile is a lightweight missile propelled by a powerful engine, delivering its payload to its target quickly and efficiently. Its high agility comes at the cost of destructive power, however, limiting its effectiveness to light, nimble fighters. Enygma Systems has developed a turret mounted Fury launcher. It comes at a steep price, but can be a powerful means of discouraging pirate fighters.</description>
```

Comment:
I think this description is more readable if a blank line is included between the description of the missile and the launcher specific description. So it displays like this:

> The Fury class interceptor missile is a lightweight missile propelled by a powerful engine, delivering its payload to its target quickly and efficiently. Its high agility comes at the cost of destructive power, however, limiting its effectiveness to light, nimble fighters.
> 
> Enygma Systems has developed a turret mounted Fury launcher. It comes at a steep price, but can be a powerful means of discouraging pirate fighters.

This change also results in the same missile description appearing in multiple places. That's generally undesirable. Since the missile `.xml` files still exist, perhaps naev could automatically display both the missile's description and the launchers description?

Reply:
I did consider programmatically including the text that way but honestly it wouldn't be a perfect solution anyhow. I can add those newlines though.

Reply:
OK, I've tweaked those descriptions. Not all in the same way, mind; some I made two separate paragraphs and some I consolidated in a different way.